project_event/models/project_task.py

```diff
@@ -162,6 +162,8 @@ def action_request(self):
     def action_option(self):
         if self.activity_task_type == 'task':
             self.draft_resources_reservation()
+        if self.task_state in ['requested', 'read', 'accepted']:
```

Comment:
@eilst I think it should be under the if self.activity_task_type == 'task' statement

Reply:
right

Reply:
right